kolibri/plugins/coach/api.py

```diff
@@ -156,7 +156,7 @@ def list(self, request, *args, **kwargs):
         Then it fetches and saves the needed information to know how many coaches
         are requesting notifications in the last five minutes
         """
-        response = super(viewsets.ReadOnlyModelViewSet, self).list(request, *args, **kwargs)
+        response = super(ClassroomNotificationsViewset, self).list(request, *args, **kwargs)
```

Comment:
have you checked this works properly?
I remember having to use the parent class because this produced an infinite loop

Reply:
I can revert this - I wanted to only do low risk code quality fixes here, so if this is at all risky, we can leave it as is.

Reply:
This was risky, and did exactly what you said. I have reverted the change, and added a comment to prevent similar foolhardiness in the future!